manifests/jetty.pp

```diff
@@ -50,26 +52,30 @@
   file { '/var/log/jetty':
     ensure => 'link',
     target => "${jetty_home}/jetty/logs",
-  } ->
```

Comment:
Grr. I lost a comment somehow. The following resource was moved into an if block. I've added ordering back in.